src/main.ts

```diff
@@ -3,48 +3,43 @@ import 'core-js/stable' // to polyfill ECMAScript features
 import 'regenerator-runtime/runtime' // to use transpiled generator functions
 
 // Vue Libraries
-import Vue from 'vue'
-import Vuetify from 'vuetify'
+import { createApp } from 'vue'
 import Vuelidate from 'vuelidate'
 import { getVueRouter } from '@/router'
-import { getPiniaStore, getVuexStore } from '@/store'
+import { getPiniaStore } from '@/store'
 import Affix from 'vue-affix'
-import Vue2Filters from 'vue2-filters' // needed by SbcFeeSummary
 import * as Sentry from '@sentry/vue'
 import VueObserveVisibility from 'vue-observe-visibility' // added to help with rendering of text area heights properly
 import Hotjar from 'vue-hotjar'
+import vuetify from './plugins/vuetify'
 
 // Base App
 // NB: must come before style imports
 import App from '@/App.vue'
 
-// Styles
-// NB: order matters - do not change
-import 'vuetify/dist/vuetify.min.css'
-import '@mdi/font/css/materialdesignicons.min.css' // ensure you are using css-loader
-import '@/assets/styles/base.scss'
-import '@/assets/styles/layout.scss'
-import '@/assets/styles/overrides.scss'
-
 // Helpers
 import { FetchConfig, GetFeatureFlag, InitLdClient, Navigate } from '@/utils'
 import KeycloakService from 'sbc-common-components/src/services/keycloak.services'
 
 // get rid of "You are running Vue in development mode" console message
-Vue.config.productionTip = false
+// https://v3-migration.vuejs.org/breaking-changes/global-api.html#config-productiontip-removed
+// Vue.config.productionTip = false
 
-Vue.use(Vuetify)
-Vue.use(Affix)
-Vue.use(Vuelidate)
-Vue.use(Vue2Filters)
-Vue.use(VueObserveVisibility)
+// Vue.use(Vuetify)
+// Vue.use(Affix)
+// Vue.use(Vuelidate)
+// Vue.use(Vue2Filters)
+// Vue.use(VueObserveVisibility)
```

Comment:
Can delete later. For now, I'm keeping these comments for reference.

Reply:
bcgov/entity#13651
bcgov/entity#13654
bcgov/entity#13653
bcgov/entity#18121